### PR TITLE
Flutter and Dart version from registry

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: src/wizard/dart/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Get the SDK from [pub.dev](https://pub.dev/packages/sentry) by adding the follow
 
 ```yml {filename:pubspec.yaml}
 dependencies:
-  sentry: ^4.0.4
+  sentry: ^{{ packages.version('sentry.dart') }}
```
Comment (marandaneto) [inline src/wizard/dart/index.md:12]: afaik it does not work for wizard, right?

**Review by marandaneto — APPROVED** [review]
(L)

File: src/wizard/dart/index.md (hunk 1 above)
Comment (bruno-garcia) [reply src/wizard/dart/index.md:12]: @marandaneto works, we have on all wizards

**Comment by bruno-garcia** [comment]
:thanks:

**Comment by philipphofmann** [comment]
Nice thanks @bruno-garcia 🚀

**Pull request merged**